test/unit/vl-map-layer-switcher.test.html

```diff
@@ -62,6 +79,16 @@
         });
       });
 
+      test('non-direct child elementen van de vl-map worden niet meegenomen in de default layer switcher', (done) => {
+        const map = fixture('vl-map-layer-switcher-layer-non-direct-child-fixture');
+        const layerSwitcher = map.querySelector('vl-map-layer-switcher');
+        map.ready.then(() => {
```

Comment:
kunt de test ook async maken en `await map.ready` doen

Reply:
Andere testen in die file deden het ook al zo (maar dat is op zich geen excuus natuurlijk)

Reply:
K zal t rap aanpassen

Reply:
Afhankelijk van developer wordt dat anders gedaan. Het is moeilijk om een bepaalde stijl af te dwingen bij al de afnemers. Zowel `done()` als promise return als `await` is prima, dus gebruik gerust wat jullie intern afgesproken hebben.